### PR TITLE
Move accessibility js route to static js and initial state

### PR DESCRIPTION
**Pull request by rullzer**
Yet again another js file that doesn't have to be served via nextcloud.
But can just be done via the webbrowser.

Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

File: apps/accessibility/appinfo/app.php [hunk 1]
```diff
@@ -34,4 +34,5 @@
 // 1. create the app
 // 2. generate css route and inject
 $app->injectCss();
-$app->injectJavascript();
+
+\OCP\Util::addScript('accessibility', 'accessibilityoca');
```
Comment (kesselb) [inline apps/accessibility/appinfo/app.php:38]: Where is `accessibilityoca` :thinking:

**Review by ChristophWurst — APPROVED** [review]

File: apps/accessibility/appinfo/app.php (hunk 1 above)
Comment (rullzer) [reply apps/accessibility/appinfo/app.php:38]: good question..

**Review by georgehrke — APPROVED** [review]

File: apps/accessibility/lib/Controller/AccessibilityController.php [hunk 2]
```diff
@@ -204,44 +203,6 @@ public function getCss(): DataDisplayResponse {
...
 	}
 
-	/**
-	 * @NoCSRFRequired
-	 * @PublicPage
-	 * @NoSameSiteCookieRequired
-	 *
-	 * @return DataDownloadResponse
-	 */
-	public function getJavascript(): DataDownloadResponse {
```
Comment (juliushaertl) [inline apps/accessibility/lib/Controller/AccessibilityController.php:214]: Please remove https://github.com/nextcloud/server/blob/cb057829f72c70e819f456edfadbb29d72dba832/apps/accessibility/appinfo/routes.php#L30 as well

**Review by juliusknorr — APPROVED** [review]
routes entry is still present, otherwise fine :+1:

**Pull request merged**